api/src/main/java/io/grpc/Grpc.java

```diff
@@ -124,4 +124,15 @@ private static String authorityFromHostAndPort(String host, int port) {
       throw new IllegalArgumentException("Invalid host or port: " + host + " " + port, ex);
     }
   }
+
+  /**
+   * Static factory for creating a new ServerBuilder.
+   *
+   * @param port the port to listen on
+   * @param creds the server identity
+   */
+  @ExperimentalApi("TODO")
+  public static ServerBuilder<?> newServerBuilderForPort(int port, ServerCredentials creds) {
```

Comment:
If we have plans to support binding to a specific IP address, would it make sense to add another method here, like
```
public static ServerBuilder<?> newServerBuilderForPort(String listeningIpAddress, int port, ServerCredentials creds)
```
For now we can throw "UnsupportedOperationException" for any `listeningIpAddress` that's not `IP_ANY` but we at least have the API defined now. And later we can add proper support.

Reply:
What good is the API if there are no callers and no implementors? And how do you gauge fitness, that it actually solves what is necessary and can be implemented if there are no callers and no implementors? FWIW, I'm not convinced that a string would be the way to go.

We can just add such an API when we add the API elsewhere.